tinygrad/renderer/assembly.py

```diff
@@ -216,6 +216,14 @@ def cast(a, dtype:DType, atype:DType, bitcast=False, u=None, pred=False):
         if lang.load_global:
           dt = dtypes.ulong if dtype.__class__ == PtrDType else dtype
           kk(*lang.render_load(args[1], ssa(u, 'dat', dtype=lang.types[dt]), dt, ss=".param"))
+      elif uop is UOps.WMMA:
+        wmma=[]
+        for v in vin[:2]:
+          for i in range(0, len(r[v]), 2): # type: ignore
```

Comment:
why `type: ignore`?

Reply:
Not sure why but mypy thinks v is of type string and throws `error: Invalid index type "str" for "dict[UOp, list[str] | str]"; expected type "UOp"`

Reply:
This has to be understood and fixed before we can merge this.

Reply:
hmm, seems like changing the var name form `v` to `vv` fixes this, not sure why `v` breaks mypy